cdk/jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
We should add the GuCDK mocks too - https://github.com/guardian/cdk/blob/main/docs/001-general-usage.md#-mocking-gucdkversion.